server/fleet/hosts.go

```diff
@@ -1207,3 +1207,16 @@ func ValidateHostScriptContents(s string) error {
 
 	return nil
 }
+
+type ScriptResult struct {
```

Comment:
Could we use `HostScriptResult` in lieu of this?

Reply:
good catch! already removed, seems like it was a leftover from a previous implementation.